build/rust/build_rust_toolchain_aux.py

```diff
@@ -77,32 +77,36 @@ def run_xpy():
     subprocess.check_call(args)
 
 
-def platform_prefix():
+def package_name():
     if sys.platform == 'darwin':
         if platform.machine() == 'arm64':
-            return 'mac-arm64'
-        return 'mac'
-    if sys.platform == 'win32':
-        return 'win'
-    return 'linux-x64'
+            platform_prefix = 'mac-arm64'
+        else:
+            platform_prefix = 'mac'
+    elif sys.platform == 'win32':
+        platform_prefix = 'win'
+    else:
+        platform_prefix = 'linux-x64'
```

Comment:
How come linux has `-x64` but mac and win don't? Can we standardize?

Reply:
Comes from Chromium's naming convention in [DEPS](https://source.chromium.org/chromium/chromium/src/+/37b3c0480f8987b116d9c8240713d869afb03d45:DEPS;l=961-988).